app/main.py

```diff
@@ -2,24 +2,62 @@
 
 
 class IntegerRange:
-    pass
+    """Data validation descriptor"""
+
+    def __init__(self, min_value: int, max_value: int) -> None:
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __set_name__(self, instance: object, name: str) -> None:
+        self.private_name = f"_{name}"
+
+    def __get__(self, instance: object, owner: object) -> str:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"{self.private_name} must be integer!")
+        if not self.min_value <= value <= self.max_value:
+            raise ValueError(f"Value must be in range from "
+                             f"{self.min_value} to {self.max_value}")
+        setattr(instance, self.private_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
In the Visitor class, the constructor (__init__ method) is missing a type hint for the return type. It should be `None`. Also, you should use the `IntegerRange` class to validate the age, weight, and height parameters.